pkgs/development/python-modules/mdformat-wikilink/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  mdformat,
+  poetry-core,
+  pytest-cov,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "mdformat-wikilink";
+  version = "0.2.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "tmr232";
+    repo = "mdformat-wikilink";
+    rev = "v${version}";
+    hash = "sha256-KOPh9iZfb3GCvslQeYBgqNaOyqtWi2llkaiWE7nmcJo=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+    pytest-cov
```

Comment:
```suggestion
```
`pytest-cov` is a pytest-plugin for tests, so we don't need it in `nativeBuildInputs`.
Also, we don't want to measure coverage because we test for packaging correctness.

Reply:
Seeing that there might be an issue with this as reported by the darwin build. Do you have any suggestions?

Reply:
Please remove `--cov=...` with substituteInPlace in postPatch.

Reply:
@natsukium apologies for ignorance. Not seeing `substituteInPlace` nor `postPatch` being touched in the files I've modified. Is this occurring at some higher module?